### PR TITLE
Update netlify

### PR DESCRIPTION
**Pull request by shiyuhang0**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/integrate-tidbcloud-with-netlify.md [hunk 1]
```diff
@@ -222,4 +222,34 @@ For a TiDB Dedicated cluster, you can get the connection string only from the Ti
     netlify deploy --prod --trigger
     ```
 
-    Go to your Netlify console to check the deployment state. After the deployment is done, the site for the app will have a public IP address provided by Netlify so that everyone can access it.
\ No newline at end of file
+    Go to your Netlify console to check the deployment state. After the deployment is done, the site for the app will have a public IP address provided by Netlify so that everyone can access it.
+
+## Using the edge function
```
Comment (ran-huang) [inline tidb-cloud/integrate-tidbcloud-with-netlify.md:227]: use edge function 和这篇文档里原有的 deploy app 的关系是什么

File: tidb-cloud/integrate-tidbcloud-with-netlify.md [hunk 2]
```diff
@@ -222,4 +222,34 @@ For a TiDB Dedicated cluster, you can get the connection string only from the Ti
...
+    import { connect } from 'https://esm.sh/@tidbcloud/serverless'
+    
+    export default async () => {
+      const conn = connect({url: Netlify.env.get('TIDB_DATABASE_URL')})
+      const result = await conn.execute('show databases')
+      return new Response(JSON.stringify(result));
+    }
+    ```
+
+3. Set the `TIDB_DATABASE_URL` environment variables
```
Comment (ran-huang) [inline tidb-cloud/integrate-tidbcloud-with-netlify.md:245]:
```suggestion
3. Set the `TIDB_DATABASE_URL` environment variables:
```

Comment (ran-huang) [reply tidb-cloud/integrate-tidbcloud-with-netlify.md:245]: 这里是在自己的本地执行吗？

Comment (shiyuhang0) [reply tidb-cloud/integrate-tidbcloud-with-netlify.md:245]: 不是，这个环境变量会被推到 netlify

File: tidb-cloud/integrate-tidbcloud-with-netlify.md (hunk 1 above)
Comment (shiyuhang0) [reply tidb-cloud/integrate-tidbcloud-with-netlify.md:227]: 关系不大，原有的 deploy app  部署上去是用的 serverless function，这里介绍了怎么用 edge function

Comment (shiyuhang0) [reply tidb-cloud/integrate-tidbcloud-with-netlify.md:227]: 我加了一句话过渡

Comment (ran-huang) [reply tidb-cloud/integrate-tidbcloud-with-netlify.md:227]: 建议在文档开头解释一下吧，"这篇文档包含了两种 xxx“

Comment (shiyuhang0) [reply tidb-cloud/integrate-tidbcloud-with-netlify.md:227]: done

File: tidb-cloud/integrate-tidbcloud-with-netlify.md [hunk 3]
```diff
@@ -222,4 +222,34 @@ For a TiDB Dedicated cluster, you can get the connection string only from the Ti
...
 
-    Go to your Netlify console to check the deployment state. After the deployment is done, the site for the app will have a public IP address provided by Netlify so that everyone can access it.
\ No newline at end of file
+    Go to your Netlify console to check the deployment state. After the deployment is done, the site for the app will have a public IP address provided by Netlify so that everyone can access it.
+
+## Using the edge function
+
+The example app above will run on the Netlify serverless function. This section shows you how to use the edge function with [TiDB Cloud serverless driver](/tidb-cloud/serverless-driver.md). The edge function is a feature provided by Netlify, it allows you to run serverless functions on the edge of the Netlify CDN.
+
+1. To use the edge function, create a directory named `netlify/edge-functions` in the root directory of your project.
```
Comment (ran-huang) [inline tidb-cloud/integrate-tidbcloud-with-netlify.md:231]:
```suggestion
1. Create a directory named `netlify/edge-functions` in the root directory of your project. 
```

File: tidb-cloud/integrate-tidbcloud-with-netlify.md [hunk 4]
```diff
@@ -222,4 +222,34 @@ For a TiDB Dedicated cluster, you can get the connection string only from the Ti
...
+      const conn = connect({url: Netlify.env.get('TIDB_DATABASE_URL')})
+      const result = await conn.execute('show databases')
+      return new Response(JSON.stringify(result));
+    }
+    ```
+
+3. Set the `TIDB_DATABASE_URL` environment variables:
+
+    ```shell
+    netlify env:set TIDB_DATABASE_URL 'mysql://<username>:<password>@<host>/<database>'
```
Comment (ran-huang) [inline tidb-cloud/integrate-tidbcloud-with-netlify.md:248]: `<username>:<password>@<host>/<database>` 这几个值是在 cloud console 的 connect 里获取吗？

File: tidb-cloud/integrate-tidbcloud-with-netlify.md [hunk 5]
```diff
@@ -222,4 +222,34 @@ For a TiDB Dedicated cluster, you can get the connection string only from the Ti
...
+    ```
+
+4. Deploy the edge function to Netlify.
+
+    ```shell
+    netlify deploy --prod --trigger
+    ```
```
Comment (ran-huang) [inline tidb-cloud/integrate-tidbcloud-with-netlify.md:255]: deploy 之后用户要怎么使用呢？这里能否提供一个使用方法，或者验证是否 deploy 成功的命令？

File: tidb-cloud/integrate-tidbcloud-with-netlify.md (hunk 4 above)
Comment (shiyuhang0) [reply tidb-cloud/integrate-tidbcloud-with-netlify.md:248]: 是的

File: tidb-cloud/integrate-tidbcloud-with-netlify.md (hunk 5 above)
Comment (shiyuhang0) [reply tidb-cloud/integrate-tidbcloud-with-netlify.md:257]: done

**Review by ran-huang — APPROVED** [review]
rest LGTM

File: tidb-cloud/integrate-tidbcloud-with-netlify.md [hunk 6]
```diff
@@ -222,4 +222,40 @@ For a TiDB Dedicated cluster, you can get the connection string only from the Ti
...
+
+1. Create a directory named `netlify/edge-functions` in the root directory of your project. 
+
+2. Create a file named `hello.ts` in the directory and add the following code:
+
+    ```typescript
+    import { connect } from 'https://esm.sh/@tidbcloud/serverless'
+    
+    export default async () => {
+      const conn = connect({url: Netlify.env.get('TIDB_DATABASE_URL')})
```
Comment (shiyuhang0) [inline tidb-cloud/integrate-tidbcloud-with-netlify.md:241]:
```suggestion
      const conn = connect({url: Netlify.env.get('DATABASE_URL')})
```

File: tidb-cloud/integrate-tidbcloud-with-netlify.md [hunk 7]
```diff
@@ -222,4 +222,40 @@ For a TiDB Dedicated cluster, you can get the connection string only from the Ti
...
+      const result = await conn.execute('show databases')
+      return new Response(JSON.stringify(result));
+    }
+   
+    export const config = { path: "/api/hello" };
+    ```
+
+3. Set the `TIDB_DATABASE_URL` environment variables. You can get the connection information from the [TiDB Cloud console](https://tidbcloud.com/).
```
Comment (shiyuhang0) [inline tidb-cloud/integrate-tidbcloud-with-netlify.md:249]:
```suggestion
3. Set the `DATABASE_URL` environment variables. You can get the connection information from the [TiDB Cloud console](https://tidbcloud.com/).
```

File: tidb-cloud/integrate-tidbcloud-with-netlify.md [hunk 8]
```diff
@@ -222,4 +222,40 @@ For a TiDB Dedicated cluster, you can get the connection string only from the Ti
...
+
+    ```shell
+    netlify env:set TIDB_DATABASE_URL 'mysql://<username>:<password>@<host>/<database>'
```
Comment (shiyuhang0) [inline tidb-cloud/integrate-tidbcloud-with-netlify.md:252]:
```suggestion
    netlify env:set DATABASE_URL 'mysql://<username>:<password>@<host>/<database>'
```

**Review by zhangyangyu — APPROVED** [review]

**Pull request merged**